extensions/androidx-paging3/build.gradle

```diff
@@ -53,7 +53,6 @@ kotlin {
     jvmTest {
       dependencies {
         implementation projects.drivers.sqliteDriver
-        implementation libs.androidx.paging3.runtime
```

Comment:
This is an aar not a jar. Not sure how this ever worked, but it's not even used here.